src/components/DpButton/DpButton.vue

```diff
@@ -11,7 +11,7 @@
       v-if="icon"
       aria-hidden="true"
       :icon="icon"
-      size="small" />
+      :size="iconSize" />
```

Comment:
I'd suggest to also enable the second icon (that after the text) to be dynamically sized. I guess it is ok if we do not support two icon sizes within one button.

Reply:
Sure. good idea. 37965ebb33a7b063733091163747fc1859e74a1a